tensor/ap_test.go

```diff
@@ -385,7 +445,9 @@ func TestUntransposeIndex(t *testing.T) {
 	}
 }
 
-func TestBroadcastStrides(t *testing.T) {
+// TODO: This test fails, why?
```

Comment:
Not failing anymore. Must have written the comment in the middle of some changes and broken something.